modules/tensor_mechanics/doc/content/source/materials/ComputeElasticityTensor.md

```diff
@@ -87,17 +87,38 @@ C_{ijkl}^{orthotropic} = \begin{bmatrix}
               \end{bmatrix}
 \end{equation}
 
+The user can also select the fill method `orthotropic`, which generates an elasticity tensor based directly on material parameters. That is, the elasticity tensor is computed via moduli of elasticity, Poisson ratios, and shear stiffnesses, see [eq:orhotropic_fill_method]
+\begin{equation}
+\label{eq:orhotropic_fill_method}
+C_{ijkl}^{orthotropic} = \begin{bmatrix}
+              E_{1} (1 - \nu_{23} \nu_{32})/k & E_{1} (\nu_{23} \nu_{31} + \nu_{21})/k & E_{1} (\nu_{21} \nu_{32} + \nu_{31})/k  &      0 &      0 &      0 \\
+              E_{2} (\nu_{13} \nu_{32} + \nu_{12})/k & E_{2} (1 - \nu_{13} \nu_{31})/k & E_{2} (\nu_{12} \nu_{31} + \nu_{32})/k &      0 &      0 &      0 \\
+              E_{3} (\nu_{12} \nu_{23} + \nu_{13})/k & E_{3} (\nu_{13} \nu_{21} + \nu_{23})/k & E_{3} (1 - \nu_{12} \nu_{21})/k &      0 &      0 &      0 \\
+                   0 &      0 &      0 & G_{12} &      0 &      0 \\
+                   0 &      0 &      0 &      0 & G_{31} &      0 \\
+                   0 &      0 &      0 &      0 &      0 & G_{23}
+              \end{bmatrix}
+\end{equation}
+where $k = 1 - \nu_{12} \nu_{21} - \nu_{23} \nu_{32} - \nu_{31} \nu_{13} - \nu_{12} \nu_{23} \nu_{31} - \nu_{21} \nu_{32} \nu_{13}$.
+ 
+
 ### Example Input File Syntax
 
 !listing modules/tensor_mechanics/test/tests/finite_strain_elastic/finite_strain_elastic_new_test.i block=Materials/elasticity_tensor
 
 In the Einstein index notation shown in [eq:rank4tensor_aux_indices], the parameter `C_ijkl`
 expects the elasticity components in the order `C_ijkl = '1111 1122 1133 2222 2233 3333 2323 3131
-1212'` for the `symmetric9` fill method option.
+1212'` for the `symmetric9` fill method option. Note that, in this case, the method `symmetric9` is used
+to enter an isotropic elasticity tensor.
+
+!listing modules/tensor_mechanics/test/tests/finite_strain_elastic_anisotropy/3d_bar_orthotropic_full_rotation.i block=Materials/elasticity_tensor
+
+For the `orthotropic` fill method, the material parameters need to be referred to the global frame and introduced in the following order: $E_{11}\;E_{22}\;E_{33}\;G_{12}\;G_{23}\;G_{31}\;\nu_{21}\;\nu_{31}\;\nu_{32}\;\nu_{12}\;\nu_{13}\;\nu_{23}$.
```

Comment:
What if the axes of orthotropy aren't in the global coordinate system? Is there a way to rotate them? I assume there must be for all of these types of materials -- I just haven't ever worked with them.

Reply:
I did not know this, but it seems this is already taken care of in `ComputeElasticityTensor` through Euler angles in the constructor:

https://github.com/idaholab/moose/blob/cdb85d6a2e2dda7f1db22c7f9e7f047aa53831ac/modules/tensor_mechanics/src/materials/ComputeElasticityTensor.C#L43

I'll add a line to the documentation